HaikuPorter/DependencyResolver.py

```diff
@@ -7,7 +7,7 @@
 
 import os
 import re
-from subprocess import CalledProcessError, check_output
+from subprocess import CalledProcessError, check_output, PIPE
```

Comment:
Probably we shouldn't pollute global namespace, we should just import subprocess and then use it as `subprocess.PIPE`.